plone/supermodel/schema.rst

```diff
@@ -330,7 +330,7 @@ tag to wrap a sequence of fields.
     ...             <max_length>10000</max_length>
     ...         </field>
     ...
-    ...         <fieldset name="dates" label="Dates" description="Standard dates">
+    ...         <fieldset name="dates" label="Dates" description="Standard dates" order="1">
```

Comment:
Hint: this is not HTML!  But XML.



Am 06. Mai 2016 10:26:31 nachm. schrieb Johannes Raggam 
<notifications@github.com>:

>> @@ -330,7 +330,7 @@ tag to wrap a sequence of fields.
>>      ...             <max_length>10000</max_length>
>>      ...         </field>
>>      ...
>> -    ...         <fieldset name="dates" label="Dates" description="Standard 
>> dates">
>> +    ...         <fieldset name="dates" label="Dates" description="Standard 
>> dates" order="1">
>
> There is no order attribute, unfortunately. See: 
> https://developer.mozilla.org/de/docs/Web/HTML/Element/fieldset
> You could use ``data-order``, if necessary.
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/plone/plone.supermodel/pull/15/files/ded88c2f04077d4665e9822671fa1cc453f7f57e#r62386245